src/content/1.7/development/form-types/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Form Types Reference
+menuTitle: Form Types Reference
+weight: 20
+---
+
+## Form types reference
+
+While developing on PrestaShop you can use any form field type that is provided by Symfony framework.
+More about Symfony field types: https://symfony.com/doc/current/reference/forms/types.html.
+In addition, PrestaShop has some custom reusable field types located in 
+`src/PrestaShopBundle/Form/Admin/Type` that should save your time:
+
+#### Text fields
+
+* [FormattedTextareaType](formatted-textarea)
+
+* [GeneratableTextType](generatable-text)
+
+* [IpAddressType](ip-address)
+
+* [ResizableTextType](resizable-text)
+    
+* [TextWithLengthCounterType](text-with-length-counter)
+
+* [TextWithUnitType](text-with-unit)
+
+* [TranslatableType](translatable)
+
+* [TranslateTextType](translate-text)
```

Comment:
`TranslateTextType` is deprecated, should we still document it?

Reply:
I guess not, ill remove it.